src/components/Books.js

```diff
@@ -28,30 +31,66 @@ const BookList = () => {
 
   return (
     <div>
-      <h3>Book List:</h3>
       <ul>
         {books.map((book) => (
-          <li key={book.item_id}>
-            <p>
-              Title:
-              {' '}
-              {book.title}
-            </p>
-            <p>
-              Author:
-              {' '}
-              {book.author}
-            </p>
-            <p>
-              Category:
-              {' '}
-              {book.category}
-            </p>
-            <button type="button" onClick={() => handleRemove(book.item_id)}>
-              remove
-            </button>
-            <button type="submit">comment</button>
-            <button type="submit">Edit</button>
+          <li key={book.item_id} className="Lesson-Panel">
+            <div>
+              <p className="School-of Text-Style-5">
+                {' '}
+                {book.category}
+              </p>
+              <p className="Title Text-Style-6">
+                {' '}
+                {book.title}
+              </p>
+              <p className="Suzanne-Collins Text-Style-8">
+                {' '}
+                {book.author}
+              </p>
+              <button type="submit" className="Comments Text-Style-8">comment</button>
```

Comment:
- [ ] According to the requirements the project should be EXACTLY the same as in the example we do this so the project is as close to real life as possible. as you can see in your books the text colour, the buttons and the completion indicator are not the same.
your project:
![afbeelding](https://user-images.githubusercontent.com/39240820/232135685-9ee927dc-f870-4265-97f6-b68afdaae755.png)
Zeplin example:
![afbeelding](https://user-images.githubusercontent.com/39240820/232135913-8d36475e-2e90-4887-9829-e1bdd2ba4ef8.png)

Reply:
**You did a great efford to this project.** 

- [ ] However, there are still places that need to be fixed. While making this requirement you created anothere issue to your books. As you can see in this screenshot books are a little out of the box in the left side. Kindly fix the width of them :)

<img width="1440" alt="Screenshot 2023-04-15 at 01 21 47" src="https://user-images.githubusercontent.com/69011963/232148421-f6cc8be8-13bb-4799-ac1d-a1e994d29b10.png">